src/Persistence/Mapping/Driver/FileDriver.php

```diff
@@ -167,13 +149,11 @@ abstract protected function loadMappingFile(string $file);
      * necessary. This may not be relevant to scenarios where caching of
      * metadata is in place, however hits very hard in scenarios where no
      * caching is used.
-     *
-     * @return void
      */
-    protected function initialize()
+    protected function initialize(): void
     {
         $this->classCache = [];
-        if ($this->globalBasename === null) {
+        if ($this->globalBasename === '') {
```

Comment:
Is this a bugfix? Do we need to fix this on 3.x?

Reply:
It would make sense to add this to 3.x. I haven't seen a way to set this property to `null`.